mantidproject/mantid
# IndirectDataAnalysis - Save Ascii files

The SaveAscii algorithm used in Indirect > Data Reduction> Energy Transfer has been updated to version 2 to enable easier re-loading of saved ascii files back into mantid.

**To test:**

Indirect > Data Reduction > Energy Transfer
Ensure default save directory is set
Ensure Instrument is set to `TOSCA`
Run Files: `TSC15352.raw`
Rebin String: `-2.5,0.015,3,-0.005,1000`
Press "Run"
Ensure `ASCII` checkbox is ticked
Press "Save"
Close interface, delete `tosca15352-TO96_2` workspace

Press `Ctrl`+`Shift`+`F`
Browse to default save directory
Select `tosca15352-TO96_2.dat`
Ensure file loads properly

Fixes #16419.

[Release Notes](https://github.com/mantidproject/mantid/blob/51225396354376c770de652e2cfdaa639e165210/docs/source/release/v3.9.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
